From b7e861aa1930bab79c1f5563fcc3712ae0114f68 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 30 Oct 2010 19:55:40 +0000
Subject: [RFC/PATCH 4/5] gettextize: git-revert: !fixup "Your local changes"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Gettextize two messages that I missed in 'gettextize: git-revert "Your
local changes" message'.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/revert.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/builtin/revert.c b/builtin/revert.c
index c8463d2..178b8a0 100644
--- a/builtin/revert.c
+++ b/builtin/revert.c
@@ -294,9 +294,9 @@ static NORETURN void die_dirty_index(const char *me)
 					  "Please, commit your changes or stash them to proceed."));
 		} else {
 			if (action == REVERT)
-				die("Your local changes would be overwritten by revert.\n");
+				die(_("Your local changes would be overwritten by revert.\n"));
 			else
-				die("Your local changes would be overwritten by cherry-pick.\n");
+				die(_("Your local changes would be overwritten by cherry-pick.\n"));
 		}
 	}
 }
-- 
1.7.3.2.312.ge13a7

